src/main/java/net/fabricmc/installer/server/ServerInstaller.java

```diff
@@ -61,13 +62,35 @@ public class ServerInstaller {
 	public static final String DEFAULT_LAUNCH_JAR_NAME = "fabric-server-launch.jar";
 	private static final Pattern SIGNATURE_FILE_PATTERN = Pattern.compile("META-INF/[^/]+\\.(SF|DSA|RSA|EC)");
 
-	public static void install(Path dir, LoaderVersion loaderVersion, String gameVersion, InstallerProgress progress) throws IOException {
+	public static void install(Path dir
+			, LoaderVersion loaderVersion
+			, String gameVersion
+			, InstallerProgress progress) throws IOException {
```

Comment:
This is very weird formatting, why change it? Please make sure it passes the checkstyle checks.

Reply:
Sorry, actually while I code I have a habit of wanting everything in front of my eyes and hate to scroll sideways. In the case of previous formatting, I had to scroll sideways, so I changed it.